src/views/Statistics.vue

```diff
@@ -83,12 +94,54 @@ export default {
         },
         { name: 'Areas climbed', value: Object.keys(areas).length },
       ];
-      this.loaded = true;
+    },
+
+    updateItems() {
+      if (!this.userDoc || !this.indexedData || !this.indexedData.routes ||
```

Comment:
do you still need to check `this.userDoc` here? i think it's covered by `this.climbDataLoaded` now.

Reply:
Ok, removed the check for this.userDoc.